### ISSUE TITLE
Collecting event without material sample?

### ISSUE THREAD

**Issue by cgendreau** [DINA-Web/collection-specs#24]
From James: Can a collecting event really have no material samples? The only case I can think of would be if we were to prepopulate a bunch of locations to make data capture quicker and some of them never got sampled from…

**Comment by cboelling** [comment]
The [definition of collecting event](https://github.com/DINA-Web/collection-specs#collecting-event) appears to stipulate that a collecting event is necessarily associated with a material sample.

The mentioned use case appears to be a pragmatic consideration. Keeping those separated from the data model is most often preferable.

**Comment by cgendreau** [comment]
So you would suggest to enforce the relationship between collecting event and material sample right?

CollectingEvent 1 ----- 1..* MaterialSample

**Comment by cboelling** [comment]
Taking the current definition as a starting point, I'd say yes and suggest, as English language definition of collecting event:

A collecting event is an event in which a material sample is collected.

HOWEVER, I think there are collecting events in the context of natural history collections where no material samples are involved: when what is collected are observations like those recorded in field diaries or sources. One striking example from the Museum für Naturkunde are our 120.000 [animal sound recordings](https://www.tierstimmenarchiv.de/).

In such cases there are no material samples, but some natural process (like the occurrence of an individual of a certain species at a given time and location) was sampled in the form of observations, followed by some documentation (e.g., notes or photographs or sound recordings).

If this is a relevant use case - as it seems to me - it should be accommodated in the data model, calling for a more general definition of collecting event.

**Comment by cgendreau** [comment]
Thanks @cboelling, this is very useful and interesting.

I see it as a general question about the place of observation (with digital evidence) in the collection management system. Such observation would have its own determination based on the digital evidence (as opposed to the physical specimen).

Here is one way to support it: https://github.com/DINA-Web/collection-specs/blob/issue_24/documentation/collectionDataModel.png

I think it's easier to detach the physical evidence from the digital to avoid having preparation "observation". Collecting event definition would also need to be updated.

**Comment by jmacklin** [comment]
I agree with the use case described by Christian B and believe there are definitely others like it. The model Christian G proposes to separate Material Samples from Observations (Digital from Physical) makes good sense to me and makes the model cleaner.

We would need to be able to track derivatives of the digital observations (not preparations) but I think this can be handled in the Multi-media module with meta-data fields that explain what the derivative was generated for (e.g., an image of a lion on the Savannah and a derivative image cropping out the lion to focus on a tree of interest, which then has an identification associated with it; or an audio recording of birds and a derivative recording that isolates one particular song, which is then tied to an identification...).

We would also have to make sure the UI supports an easy asynchronous ability to add Observations to a Collecting Event after a physical Material Sample has been processed. For example, I take several pictures/videos of a plant with an insect on it that is eating its leaves. I then collect the plant and the insect as Material Samples and process them. Months later, I want to add my pictures/videos related to the event that document the observation of herbivory.
